spec.bs

```diff
@@ -370,16 +371,14 @@ Payment Confirmation.
     the transaction and the merchant finishes the checkout process for the user.
 
 The sample code for authenticating the user follows. Note that the example code
-presumes access to await/async, for easier to read promise handling.
+presumes access to async/await, for easier to read promise handling.
 
 <pre class="example" id="authentication-example" highlight="js">
 /* isSecurePaymentConfirmationAvailable indicates whether the browser */
 /* supports SPC. It does not indicate whether the user has a credential */
 /* ready to go on this device. */
 const spcAvailable =
-  PaymentRequest &&
-  PaymentRequest.isSecurePaymentConfirmationAvailable &&
-  await PaymentRequest.isSecurePaymentConfirmationAvailable();
+  await PaymentRequest?.isSecurePaymentConfirmationAvailable();
```

Comment:
This is cool, I've never seen it before!

I think it's technically optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) rather than nullish coalescing (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), right?

Does this work as written if `PaymentRequest` itself is not defined? (As opposed to isSecurePaymentConfirmationAvailable not being defined). It seemed not when I tried it in Firefox, but I may have been holding it wrong. I had to do:

```
window.PaymentRequest?.isSecurePaymentConfirmationAvailable();
```

Without the `window`, I just got:

```
PaymentRequest?.isSecurePaymentConfirmationAvailable()
Uncaught ReferenceError: PaymentRequest is not defined
    <anonymous> debugger eval code:1
debugger eval code:1:1
```